For analyzing failures we need the full diff when the file checks did not find the expected strings in the source.
See https://github.com/pytorch/pytorch/pull/189410

Author: Alexander Grund (TU Dresden)
diff --git a/torch/csrc/jit/frontend/source_range.h b/torch/csrc/jit/frontend/source_range.h
--- a/torch/csrc/jit/frontend/source_range.h
+++ b/torch/csrc/jit/frontend/source_range.h
@@ -450,8 +450,8 @@ struct TORCH_API Source {
 // A SourceRange is a reference to subset of a Source, specified by `start` and
 // `end` byte offsets into the source text.
 struct TORCH_API SourceRange {
-  SourceRange(std::shared_ptr<Source> source_view, size_t start_, size_t end_)
-      : source_view_(std::move(source_view)), start_(start_), end_(end_) {
+  SourceRange(std::shared_ptr<Source> source_view, size_t start, size_t end)
+      : source_view_(std::move(source_view)), start_(start), end_(end) {
     if (source_view_) {
       start_iter_ = source_view_->text_str().iter_for_pos(start_);
     }
@@ -460,12 +460,12 @@ struct TORCH_API SourceRange {
   SourceRange() : source_view_(nullptr), start_(0), end_(0) {}
 
   SourceRange(
-      std::shared_ptr<Source> source_view_,
+      std::shared_ptr<Source> source_view,
       StringCordView::Iterator start_iter,
-      size_t end_)
-      : source_view_(std::move(source_view_)),
+      size_t end)
+      : source_view_(std::move(source_view)),
         start_(start_iter.pos()),
-        end_(end_),
+        end_(end),
         start_iter_(start_iter) {}
 
   const std::string_view token_text() const {
diff --git a/torch/csrc/jit/testing/file_check.cpp b/torch/csrc/jit/testing/file_check.cpp
--- a/torch/csrc/jit/testing/file_check.cpp
+++ b/torch/csrc/jit/testing/file_check.cpp
@@ -94,14 +94,12 @@ size_t assertFind(
     const std::function<void(std::ostream& out)>& extra_msg = nullptr) {
   auto pos = search_range.source()->text_str().find(sub, search_range.start());
   if (pos == std::string::npos || (pos + sub.size()) > search_range.end()) {
-    auto found_range =
-        SourceRange(search_range.source(), search_range.start(), sub.size());
     std::stringstream ss;
     ss << "Expected to find ";
     c10::printQuotedString(ss, sub);
     ss << " but did not find it" << '\n';
     ss << "Searched string:" << '\n';
-    found_range.highlight(ss);
+    search_range.highlight(ss);
     if (extra_msg) {
       extra_msg(ss);
     }
